spec/unit/mutest/integration_spec.rb

```diff
@@ -11,14 +11,15 @@
   end
 
   describe '.setup' do
-    subject { described_class.setup(kernel, name) }
+    subject { described_class.setup(requirer, name) }
 
-    let(:name)   { 'null'               }
-    let(:kernel) { class_double(Kernel) }
+    let(:name) { 'null' }
```

Comment:
Sure. I think I broke the alignment and then rubocop told me to fix it to this. I'll make them line up again.